.vscode/settings.json

```diff
@@ -9,11 +9,39 @@
   "[html]": {
     "editor.formatOnSave": false
   },
+  "[ejs]": {
+    "editor.formatOnSave": false
+  },
+  "[julia]": {
+    "editor.formatOnSave": false,
+    "editor.wordBasedSuggestions": "off",
+    "editor.defaultFormatter": "julialang.language-julia",
+    "editor.wordSeparators": "`~#$%^&*()-=+[{]}\\|;:'\",.<>/?"
+  },
+  "[python]": {
+    "diffEditor.ignoreTrimWhitespace": false,
+    "gitlens.codeLens.symbolScopes": ["!Module"],
+    "editor.formatOnSave": false,
+    "editor.formatOnType": false,
+    "editor.wordBasedSuggestions": "off",
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "editor.tabSize": 2,
   "deno.enable": true,
   "deno.lint": true,
   "deno.unstable": true,
-  "deno.importMap": "./src/import_map.json"
+  "deno.importMap": "./src/import_map.json",
```

Comment:
This is the settings we use, but it is overwriting the other one. 

Is the value ok ? I believe so.